test-network/docker/docker-compose-test-net.yaml

```diff
@@ -123,3 +123,25 @@ services:
       - 9051:9051
     networks:
       - test
+  
+  cli:
```

Comment:
Why start the CLI fabric-tools container with the network?
We worked hard to remove the CLI container from test-network, since we want to show people how to use commands from their local environment rather than exec into a CLI container.